src/ServiceInsight.Tests/AboutViewModelTests.cs

```diff
@@ -34,7 +34,7 @@ public void Should_fetch_service_control_version()
         {
             ((IActivate)sut).Activate();
 
-            serviceControl.Received(1).GetVersion();
+            _ = serviceControl.Received(1).GetVersion();
```

Comment:
This won't fix this test. This test was never proving anything, since the call to `GetVersion` is fire-and-forget. The correct fix is:

```C#
        [Test]
        public async Task Should_fetch_service_control_version()
        {
            ((IActivate)sut).Activate();

            await serviceControl.Received(1).GetVersion();
        }
```

or

```C#
        [Test]
        public Task Should_fetch_service_control_version()
        {
            ((IActivate)sut).Activate();

            return serviceControl.Received(1).GetVersion();
        }
```

I'll push a commit which fixes these.